README.md

```diff
@@ -126,25 +126,15 @@ pip install -e .
 
 ### Linux
 
-First, you need to install protobuf.
-
-Ubuntu users: the quickest way to install protobuf is to run
+First, you need to install protobuf. The minimum Protobuf version required by ONNX is 3.0.0.
 
+Ubuntu 22.04 (and newer) users may choose to install protobuf via
 ```bash
 apt-get install python3-pip python3-dev libprotobuf-dev protobuf-compiler
 ```
+In this case, it is recommended to add `-DONNX_USE_PROTOBUF_SHARED_LIBS=ON` to CMAKE_ARGS in the ONNX build step.
 
-Then you can build ONNX as:
-```
-export CMAKE_ARGS="-DONNX_USE_PROTOBUF_SHARED_LIBS=ON"
-git clone --recursive https://github.com/onnx/onnx.git
```

Comment:
why do we remove the following instructions?

Reply:
I removed it from here because it was duplicated information. the ONNX build steps are repeated in line 177 (167 after the changes in this PR).

Here there are 3 different ways to install protobuf (from ubuntu repos, from source in debian/ubuntu, and from source in CentOS/RHEL/Fedora). The steps on how to build ONNX from source are the next step.